subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1592,6 +1595,11 @@ static int light_ctrl_srv_start(struct bt_mesh_model *model)
 
 	atomic_set_bit(&srv->flags, FLAG_STARTED);
 
+	if (IS_ENABLED(CONFIG_BT_MESH_LIGHT_CTRL_SRV_ENABLED) &&
+	    !atomic_test_and_clear_bit(&srv->flags, FLAG_SETTINGS_SET)) {
+		atomic_set_bit(&srv->flags, FLAG_CTRL_SRV_MANUALLY_ENABLED);
```

Comment:
> This fixes Generic OnPowerUp state behavior in the sample.

Should this flag (and the functionality it enables) be removed altogether, or is there still a valid way to enable the controller before start?

Reply:
We don't have a light lc client sample and nRF Mesh mobile app, which is meant to be used for the sample testing, doesn't have such functionality either.

I can remove this option from the server and enable it manually in the sample if it is not provisioned yet. This will also work.

Reply:
Moved this functionality the sample.